HLTrigger/HLTcore/src/HLTPrescaleProvider.cc

```diff
@@ -117,7 +117,7 @@ std::pair<int, int> HLTPrescaleProvider::prescaleValues(const edm::Event& iEvent
                                                         const edm::EventSetup& iSetup,
                                                         const std::string& trigger) {
   // start with setting both L1T and HLT prescale values to 0
-  std::pair<int, int> result(std::pair<int, int>(0, 0));
+  std::pair<double, int> result(std::pair<double, int>(0, 0));
```

Comment:
The method HLTPrescaleProvider::prescaleValues then needs to be changed to return type std::pair<double, int>,
but that will wreck havoc with all clients using it!